lib/htmlValidator.js

```diff
@@ -46,31 +49,43 @@ module.exports = function(app, callback) {
           'java', ['-Xss1024k', '-cp', vnu, 'nu.validator.servlet.Main', params.htmlValidator.port ? params.htmlValidator.port : '8888']
         );
 
+        console.log(('Validator intializing...').yellow);
+
         validatorProcess.stdout.on('data', (data) => {
           if (`${data}`.includes('Initialization complete')) {
+            clearTimeout(validatorTimeout);
+            console.log(('Validator initialization completed successfully.').green);
```

Comment:
and yeah we only need one message indicating that the validator was started, so pick the line of code that makes the most sense for it to log it and use that message